tools/sof-tplgreader.py

```diff
@@ -51,7 +51,7 @@ def loadFile(self, filename, sofcard=0):
         # format pipeline, this change for script direct access 'rate' 'channel' 'dev'
         for pipeline in self._pipeline_lst:
             #pipeline['fmt']=pipeline['fmt'].upper().replace('LE', '_LE')
-            pipeline['rate'] = pipeline['rate_min']
+            pipeline['rate'] = pipeline['rate_max']
```

Comment:
I'm not sure which is better, rate_min/rate_max  
but from your comment, can you try to judgement the `rate_min` value?
So if it is 0, you setup `rate` as `rate_max`

Reply:
This is a good idea.